lib/src/week_view/week_view.dart

```diff
@@ -462,6 +462,11 @@ class WeekViewState<T extends Object?> extends State<WeekView<T>> {
 
     // Update builders and callbacks
     _assignBuilders();
+
+    if (widget.scrollOffset != oldWidget.scrollOffset) {
+      _lastScrollOffset = widget.scrollOffset;
```

Comment:
This line should be like following,

 _lastScrollOffset = widget.scrollOffset ??
        widget.startDuration.inMinutes * widget.heightPerMinute;